openshift/imageStream-template.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: revsolver-imagestream
+  annotations:
+    description: This is Thoth Core - Reverse Solver ImageStream
+    openshift.io/display-name: "Thoth: Reverse Solver ImageStream"
+    version: 0.4.0
```

Comment:
Why different versions in these template?

Reply:
copy-:spaghetti: error